### ISSUE TITLE
Issue / Bug tracking on Vocol? 

### ISSUE THREAD

**Issue by np00** [vocol/vocol#75]
To add another way for Non-IT users to provide the ontology development with valuable feedback, having some kind of issue tracking system in place could be helpful. 

One way to add it would be simply to base it on GitHub Issues, but then VoCol becomes more and more dependent on it.

**Comment by lavdim** [comment]
We will see the possibilities to integrate the issues from GitHub, in a convenient and read-only form which provides easy access to the users. But a complete integration for now it is not feasible. 

The Issue Management is left on the Repository Hosting Platforms.

**Comment by hauschke** [comment]
From my point of view it would be enough to have a list of all issues with a link to the respective issue on github/gitlab. Managing issues in Vocol is not necessary.

**Comment by np00** [comment]
Yes, I think so too, but maybe not just as another Tab with an external link, because they are getting too many. Maybe we should get in touch with Rohan to have a nice Design :)
